CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## To be released
+## v2.8.2 (May 30, 2017)
```

Comment:
This probably should not be a release. Releases are automated via Mobitron. See [release.md](https://github.com/mobify/mobify-code-style/blob/develop/RELEASE.md)